french/dessine/miel/forest/chantilly/asperge.md

```diff
@@ -0,0 +1,2 @@
+En plein sur les babines! 
+L'ours kiffe grave la chantilly - il vous fait les yeux du chat de Shrek puis vous leche le reste de miel que vous avez sur la main.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.